GPU/Common/SplineCommon.cpp

```diff
@@ -926,16 +926,19 @@ void DrawEngineCommon::SubmitSpline(const void *control_points, const void *indi
 	patch.patchFacing = patchFacing;
 
 	if (g_Config.bHardwareTessellation && g_Config.bHardwareTransform && !g_Config.bSoftwareRendering) {
-		float *pos = (float*)(decoded + 65536 * 18); // Size 3 float
-		float *tex = pos + count_u * count_v * 3; // Size 3 float
-		float *col = tex + count_u * count_v * 3; // Size 4 float
+		int stride = g_Config.iGPUBackend == GPU_BACKEND_VULKAN || g_Config.iGPUBackend == GPU_BACKEND_DIRECT3D11 ? 4 : 3;
```

Comment:
Oh, absolutely agree about parens on bitwise operators.  Those have weird precedence.  But I've never written `a == b ? c : d` and expected `a == (b ? c : d)`.

I just hate seeing code like this (which I do frequently see, together):
```c++
int foo = a * 3 + 5;
std::string name = first_name + " " + last_name;
bool verified = ((foo == 42) ? (bar == 7) : false);
```

It's like the wheels fall off their confidence train when they hit the ternary (which is probably not even the best way to write the above, which makes it even more of a perfect example - it was wrong to use, they were not confident in using it, but they awkwardly used it anyway.)  What really gets me is the extra `()` around the entire expression, which is very common... sigh.

-[Unknown]